packages/cli-repl/src/cli-repl.spec.ts

```diff
@@ -0,0 +1,10 @@
+import mapCliToDriver from './arg-mapper';
```

Comment:
If this file does nothing i'd remove it, would only make eslint unhappy once we add it